src/redisjson.rs

```diff
@@ -351,8 +360,17 @@ pub mod type_methods {
         let json = match encver {
             0 => RedisJSON {
                 data: backward::json_rdb_load(rdb),
```

Comment:
Since this function has grown, I recommend removing the `unsafe` from the function signature (which makes the whole function unsafe) and putting it just around the `backward::json_rdb_load(rdb)` which is the only unsafe thing here.

Reply:
done